pact.cabal

```diff
@@ -157,10 +157,10 @@ library
     , reflection
     , semigroupoids >= 5.0
     , servant
-    , text >= 1.2.2.1 && < 1.3
-    , transformers >= 0.5.2.0 && < 0.6
+    , text >= 1.2.2.1 && < 2
+    , transformers >= 0.5.2.0 && < 0.7
     , trifecta >= 2.1
-    , unordered-containers >= 0.2.7.2 && <= 0.2.15.0
+    , unordered-containers >= 0.2.7.2 && <= 0.2.20
```

Comment:
This change affects JSON serialization and breaks backward compatibility of hashes, even with `hashable-1.3.0.0`. :-(

Reply:
`unordered-containers-2.16.0.0` breaks backward compatibility. I think it is [this](https://github.com/haskell-unordered-containers/unordered-containers/pull/317) change the causes it.